sites/cheerpx/src/content/docs/11-guides/Networking.md

```diff
@@ -56,8 +62,10 @@ const cx = await CheerpX.Linux.create({
 
 What is happening here?
 
-- `controlUrl` is a string URL of the Tailscale control plane which verifies the user's identity. You only need to pass this option if you are [self-hosting Tailscale](/docs/guides/Networking#self-hosting-headscale).
-- `authKey` is string with an auth key to register new users/devices that are pre-authenticated. You can create an auth key [here](https://login.tailscale.com/admin/settings/keys).
+- [authKey]: A string containing an authentication key for registering pre-authenticated users or devices. You can generate one [here](https://login.tailscale.com/admin/settings/keys).
+- [controlUrl]: The Tailscale control plane URL for identity verification. This is only needed if you are [self-hosting Tailscale](/docs/guides/Networking#self-hosting-headscale).
```

Comment:
This is not accurate. The control plain also coordinates access to the network, it's not just for authentication. Reword this to be something along the lines of: "When [self-hosting Headscale] you need to provide the control plain URL of your Headscale server. By default the main Tailscale control plane is used